aws/resource_aws_appmesh_virtual_router.go

```diff
@@ -35,18 +41,16 @@ func resourceAwsAppmeshVirtualRouter() *schema.Resource {
 
 			"spec": {
 				Type:     schema.TypeList,
-				Required: true,
-				MinItems: 1,
-				MaxItems: 1,
+				Removed:  "Remove `spec` configuration block completely",
```

Comment:
Not going through `Deprecated` cycle here, straight to `Removed` as AppMesh is a preview service.

Reply:
I assume this message is surfaced to the user? Is there a way to detect values in `service_names` and instruct them to create a VirtualService resource by that name?